### ISSUE TITLE
Question about getting the `Value::Map` and `Value::Array`'s inner value by key

### ISSUE THREAD

**Issue by temeddix** [Idein/msgpack-schema#33]
```rust
let originalTree = msgpack!({
    "someKey": "someValue"
})
let bytes = serialize(originalTree);
let newTree = deserialize::<Value>(&bytes).unwrap();
let expectedValue = &newTree.as_map().unwrap().get("someKey").unwrap();
```

This code gives an error:
```
the type `[(msgpack_value::Value, msgpack_value::Value)]` cannot be indexed by `&str`
the trait `std::slice::SliceIndex<[(msgpack_value::Value, msgpack_value::Value)]>` is not implemented for `&str`rustc[Click for full compiler diagnostic](rust-analyzer-diagnostics-view:/diagnostic%20message%20%5B0%5D?0#file%3A%2F%2F%2FUsers%2Ftemeddix%2FDocuments%2FGitHub%2Fflutter-rust-app-template%2Fnative%2Fhub%2Fsrc%2Fsample_functions.rs)
sample_functions.rs(15, 41): required by a bound introduced by this call
mod.rs(603, 12): required by a bound in `core::slice::<impl [T]>::get`
```

Is there any way to overcome this?

**Comment by temeddix** [comment]
<img width="669" alt="image" src="https://github.com/Idein/msgpack-schema/assets/66480156/03a75975-64f8-4186-80bc-b924ccee49d2">

`serde_json` allows indexing into the value.

**Comment by temeddix** [comment]
@nyuichi I hope I can contribute and make a PR related to this idea if we can talk about it

**Comment by nyuichi** [comment]
@temeddix Sure, thanks for sending the pr. I'm looking into it.

**Comment by umadein** [comment]
The PR was merged
https://github.com/Idein/msgpack-schema/pull/34

**Comment by nyuichi** [comment]
@temeddix I've just released 0.8 so please check it out. Thank you for patience.

**Comment by temeddix** [comment]
That's good to hear, I would be glad to contribute to this project again later 🙂
